forms-shared/tests/sharepoint/getValuesForSharepoint.ts

```diff
@@ -206,6 +212,63 @@ describe('getValuesForFields', () => {
       expect(true).toBeFalsy()
     } catch (error) {}
   })
+
+  it('should match snapshots', () => {
+    getExampleFormPairs({
+      formDefinitionFilterFn: (formDefinition): formDefinition is FormDefinition =>
+        formDefinition.type === FormDefinitionType.SlovenskoSkGeneric &&
+        formDefinition.sharepointData !== undefined,
+    }).forEach(({ formDefinition, exampleForm }) => {
+      const result = getValuesForFields(
+        (formDefinition as FormDefinitionSlovenskoSkGeneric).sharepointData!,
+        { ginisDocumentId: 'MAG123', formDefinitionSlug: formDefinition.slug, title: 'FormTitle' },
+        exampleForm.formData,
+        fieldMap,
+      )
+      expect(result).toMatchSnapshot()
+    })
+  })
 })
 
-// TODO snapshot tests
+const fieldMap = {
```

Comment:
Why is this directly in the test? Can't this be shared between actual implementation and the test? Does the test make sense if it needs specific "instructions"?

Reply:
Where is this field map located for the actual implementation?

Reply:
The field map needs to actually be retrieved from the sharepoint, see function `getAllFieldsMappings` in sharepoint subservice. That is why it is hardcoded here in the test.